### PR TITLE
wip: switch to ProConnect

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

**Comment by davidbgk** [comment]
🌟 

Do we keep both for now?

**Comment by yohanboniface** [comment]
Not really, in the sense that only ProConnect will be exposed. I thought we could keep the possibility to rollback for a few days, just in case. But we could also just remove the MonComptePro backend, and revert it in case of urgency.

File: umap_dsfr/static/umap/theme.css [hunk 1]
```diff
@@ -2,6 +2,19 @@
     --color-darkBlue: rgb(0, 0, 145);
 }
 
+.login-proconnect {
```
Comment (yohanboniface) [inline umap_dsfr/static/umap/theme.css:5]: Is that CSS used somewhere at the end ? I did not find the MonComptePro button.

Comment (davidbgk) [reply umap_dsfr/static/umap/theme.css:5]: It's in use at https://umap.incubateur.anct.gouv.fr/en/login/ when you try to save a map without being logged in first.

Comment (davidbgk) [reply umap_dsfr/static/umap/theme.css:5]: The class is generated here https://github.com/umap-project/umap/blob/master/umap/templates/registration/login.html#L59

Comment (yohanboniface) [reply umap_dsfr/static/umap/theme.css:5]: Thanks! It works then :)

**Review by davidbgk — APPROVED** [review]

**Pull request merged**